src/Controller/EnMarche/UserController.php

```diff
@@ -196,4 +197,23 @@ public function terminateMembershipAction(Request $request): Response
             'form' => $form->createView(),
         ]);
     }
+
+    /**
+     * @Route("/chart",
+     *     name="app_user_set_accept_chart",
+     *     condition="request.isXmlHttpRequest()"
+     * )
+     * @Method("GET")
+     */
+    public function chartAcceptationAction(): JsonResponse
+    {
+        $this->getUser()->setChartAccepted(true);
+
+        $this->getDoctrine()->getManager()->flush();
```

Comment:
can you inject `ObjectManager` as an argument of the `chartAcceptationAction` method?